Controllers/HomeController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+	public class HomeController : Controller
+	{
+		[Route("")]
+		[Route("app")]
+		[Route("app/{path}")]
+		[Route("app/product/{productId}")]

```

Comment:
This route (`app/product/{productId}`) is already accounted for in the previous route (`app/{*path}`).